mscore/prefsdialog.ui

```diff
@@ -3377,6 +3377,22 @@ Adjusting latency can help synchronize your MIDI hardware with MuseScore's inter
          </layout>
         </widget>
        </item>
+       <item>
+        <widget class="QGroupBox" name="groupBox_omr">
+         <property name="title">
+          <string>OMR</string>
+         </property>
+         <layout class="QGridLayout" name="gridLayout_14">
```

Comment:
leads to a warning:  C:\MuseScore\mscore\prefsdialog.ui : warning : The name 'gridLayout_14' (QGridLayout) is already in use, defaulting to 'gridLayout_141'.

Reply:
Thanks! fixed